src/internal/ExpandTransitionChild.js

```diff
@@ -32,38 +34,35 @@ class ExpandTransitionChild extends Component {
   }
 
   componentWillEnter(callback) {
-    const {enterDelay} = this.props;
+    const {transitionDelay} = this.props;
     const {style} = ReactDOM.findDOMNode(this);
     style.height = 0;
-
-    if (enterDelay) {
-      this.enterTimer = setTimeout(() => callback(), 450);
-      return;
-    }
-
-    callback();
+    this.enterTimer = setTimeout(() => callback(), transitionDelay);
   }
 
   componentDidEnter() {
     this.open();
   }
 
   componentWillLeave(callback) {
-    const style = ReactDOM.findDOMNode(this).style;
+    const {transitionDuration} = this.props;
+    const {style} = ReactDOM.findDOMNode(this);
     style.height = this.refs.wrapper.clientHeight;
```

Comment:
Hmm. I'll test. I think not. It may be a line left over from where I was trying to fix an issue by setting the height to auto once it has expanded.

It wasn't quite working with the callbacks available in the react transition group API. I was trying to avoid adding any more timers. So planning on revisiting soon (the transition component) to see if I can fit that fix in a bit better.